astronomer/providers/apache/hive/example_dags/example_hive.py

```diff
@@ -198,6 +191,11 @@ def add_inbound_rule_for_security_group(task_instance: Any) -> None:
                 }
             ],
         )
+    except ClientError as error:
+        if error.response.get("Error", {}).get("Code", "") == BOTO_DUPLICATE_PERMISSION_ERROR:
+            logging.info("Ingress for port 22 already authorized")
```

Comment:
what about logging the actual error messages? it should be in the error object.

Reply:
Addressed. Included in the log.